packages/bot-web-ui/src/assets/sass/run-panel.scss

```diff
@@ -30,4 +29,33 @@
         height: 64px;
         margin: 0 16px;
     }
+    &-tab {
+        &__content {
+            height: calc(100vh - 310.39px);
```

Comment:
Could we create a variable in `app.scss` for this, it'll be more clear as to what is being calculated here.

Reply:
I'm gonna re-evaluate what's going on here. I completely forgot. 😅